docs/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md

```diff
@@ -171,9 +115,9 @@ For each Content-Type, the following endpoints are automatically generated:
 ::::
 :::::
 
-### Unified response format
+## Requests
 
-Whatever the query, the response is always an object with the following keys:
+No matter the request, the response is always an object with the following keys:
```

Comment:
This isn't true, it only applies to core logic. All plugins (be it community or even our own) may not and are not required to follow this format.

Examples being both users-permissions and upload plugins do not follow this format. From the user perspective these are core endpoints but they are actually plugins.

Might be worth a callout here.

Reply:
This has changed since RFCs were written, then (and I know it caused some issues for the users 😅).
Thanks for the heads-up, I've just updated it.